README.md

```diff
@@ -20,7 +24,20 @@ Dependencies not listed in the `setup.py` are tabulated below. We are referring
 * [mpi4py](https://mpi4py.readthedocs.io/en/stable/install.html) for distributed training
 * [horovod](https://github.com/horovod/horovod#install) for distributed training
 
-We prepared a script to install the `torch-geometric`, which can be executed as `install_geometric.sh cu101 1.6.0` where the first argument `cu101` is the CUDA version and the second is the `pytorch` version.
+We prepared a script to install the `torch-geometric`, which can be executed as `install_geometric.sh cu113 1.11.0` where the first argument `cu113` is the CUDA version and the second is the `pytorch` version.
+
+You can find out the cuda and pytorch version by running 
+
+```
+python -c "import torch; print(torch.__version__, torch.version.cuda)"
+```
+and then run 
+```
+export $CUDA_VERSION="your-cuda-version"
+export $TORCH_VERSION="your-torch-version"
+install_geometric.sh $CUDA_VERSION $TORCH_VERSION
+```
+where the cuda version must be in cu1xx format.
 
```

Comment:
Add a short list of versioning number of some main packages. (pytorch, torch-lightning, faiss-gpu, cuda).